2003-02-06  Jakub Jelinek  <jakub@redhat.com>

	* emulparams/elf64_s390.sh (LIBPATH_SUFFIX): Set to 64.
	* emulparams/elf_x86_64.sh (LIBPATH_SUFFIX): Likewise.
	* emulparams/elf64_sparc.sh (LIBPATH_SUFFIX): Set instead of suffix.

--- ld/emulparams/elf64_s390.sh.jj	2003-01-08 10:02:43.000000000 -0500
+++ ld/emulparams/elf64_s390.sh	2003-02-06 09:37:12.000000000 -0500
@@ -26,7 +26,7 @@ case "$target" in
   s390*-linux*)
     case "$EMULATION_NAME" in
       *64*)
-	LIBPATH_SUFFIX=$suffix ;;
+	LIBPATH_SUFFIX=64 ;;
     esac
     ;;
 esac
--- ld/emulparams/elf_x86_64.sh.jj	2003-01-08 10:02:44.000000000 -0500
+++ ld/emulparams/elf_x86_64.sh	2003-02-06 09:37:38.000000000 -0500
@@ -24,7 +24,7 @@ fi
 case "$target" in
   x86_64*-linux*)
     case "$EMULATION_NAME" in
-      *64*) LIBPATH_SUFFIX=$suffix ;;
+      *64*) LIBPATH_SUFFIX=64 ;;
     esac
     ;;
 esac
--- ld/emulparams/elf64_sparc.sh.jj	2003-01-08 10:02:44.000000000 -0500
+++ ld/emulparams/elf64_sparc.sh	2003-02-06 09:39:50.000000000 -0500
@@ -42,9 +42,9 @@ case "$EMULATION_NAME" in
   *64*)
     case "$target" in
       sparc*-linux*)
-	suffix=64 ;;
+	LIBPATH_SUFFIX=64 ;;
       sparc*-solaris*)
-	suffix=/sparcv9 ;;
+	LIBPATH_SUFFIX=/sparcv9 ;;
     esac
     ;;
 esac

